server/datastore/mysql/software.go

```diff
@@ -119,23 +119,58 @@ func nothingChanged(current, incoming []fleet.Software, minLastOpenedAtDiff time
 	return true
 }
 
+func (ds *Datastore) ListSoftwareByHostIDShort(ctx context.Context, hostID uint) ([]fleet.Software, error) {
+	return listSoftwareByHostIDShort(ctx, ds.reader, hostID)
+}
+
+func listSoftwareByHostIDShort(
+	ctx context.Context,
+	db sqlx.QueryerContext,
+	hostID uint,
+) ([]fleet.Software, error) {
+	q := `
+SELECT
+    s.id,
+    s.name,
+    s.version,
+    s.source,
+    s.bundle_identifier,
+    s.release,
+    s.version,
```

Comment:
`s.vendor,`

Reply:
nice catch

Reply:
Sorry, I meant replace this occurence of `s.version,` with s.vendor, (`s.version` is already on the third column, line 135)

Reply:
fixed